core/workspace_comment_svg.js

```diff
@@ -46,6 +46,30 @@ goog.require('Blockly.Events.CommentMove');
 goog.require('Blockly.Events.Selected');
 
 
+/**
+ * Size of the resize icon.
+ * @type {number}
+ * @const
+ * @private
```

Comment:
No longer needs private annotation--it's private by virtue of not being exported.